plugins/inputs/windows_event_log/wineventlog/wineventlog_test.go

```diff
@@ -0,0 +1,163 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build windows
+// +build windows
+
+package wineventlog
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/sys/windows/svc/eventlog"
+)
+
+var (
+	// common inputs for creating an EventLog.
+	NAME = "Application"
+	// 2 is ERROR
+	LEVELS          = []string{"2"}
+	GROUP_NAME      = "fake"
+	STREAM_NAME     = "fake"
+	RENDER_FMT      = FormatPlainText
+	DEST            = "fake"
+	STATE_FILE_PATH = "fake"
+	BATCH_SIZE      = 99
```

Comment:
There's nothing that would require us to test different batch size scenarios, right? At least, not related to this change - just wanted to ask to be sure.

Reply:
Correct, this PR is not changing the hard coded batch size, `BatchReadSizeValue = 170`.

There is another open issue related to choosing the right batch size:
https://github.com/aws/amazon-cloudwatch-agent/issues/231